server/base/application.ts

```diff
@@ -16,22 +16,22 @@
 
 import * as express from 'express';
 
-export default class Application {
-  private static app_: express.Application = express();
+export class Application {
+  private static app: express.Application = express();
 
-  public static async start() {
+  public static async START() {
```

Comment:
Thank you for this change!
BTW, it seems a little bit weird to me.
Could you explain why this function should be upper case name? (including startForTesting and route)
Is that also a kind of lint error? If so, we might have to disable the rule.

Reply:
I changed it to uppercase because the lint error caused the static method name to be in upper case. (lint option: function-name)

Reply:
Thank you for detailed explanation.
I'll re-check it again after merging this patch :)